src/components/CellWithPopover/CellWithPopover.tsx

```diff
@@ -0,0 +1,26 @@
+import cn from 'bem-cn-lite';
+
+import {Popover, type PopoverProps} from '@gravity-ui/uikit';
+
+import './CellWithPopover.scss';
+
+const b = cn('ydb-cell-with-popover');
+
+interface CellWithPopoverProps extends PopoverProps {
+    wrapperClassName?: string;
+}
+
+export function CellWithPopover({
+    children,
+    className,
+    wrapperClassName,
+    ...props
+}: CellWithPopoverProps) {
+    return (
+        <div className={b('wrapper', wrapperClassName)}>
+            <Popover className={b(null, className)} {...props}>
+                {children}
+            </Popover>
+        </div>
```

Comment:
It better to put b-style higher. So you have `block__element` inside `block`. It will be more correct in terms of BEM methodology. More specific styles should be further in components tree